src/cpip-pack

```diff
@@ -602,8 +612,8 @@ if [[ $poetry_dir ]]; then
     echo "$info" "installing pip packages via Poetry..."
     cd "$poetry_dir"
     save_poetry_lockfile
-    [[ $unlock ]] && poetry update --lock $no_dev $quiet $ansi $no_ansi
-    poetry install $no_dev $quiet $ansi $no_ansi
+    [[ $unlock ]] && poetry update --lock $no_dev $extras $quiet $ansi $no_ansi
```

Comment:
remove the `$extras` from this line

`poetry update` does not accept `$extras` argument... I assume since `poetry update` does not include that option, it will update the extras regardless

Reply:
:blush: nice catch! sorry assuming poetry would have kept `--extras` and `--no-dev` consistent